bin/cli.ml

```diff
@@ -53,6 +53,13 @@ let opam =
   let docv = "FILE" in
   Arg.(value & opt (some path_arg) None & info [ "opam" ] ~doc ~docv)
 
+let token =
+  let doc =
+    "The github token to use. If absent looks for a token stored locally."
```

Comment:
Maybe we should clarify that it's okay not to pass a token through this option and that's actually the regular usecase. How about mentioning this is used to replace or override the regular config token instead?

Reply:
done